assignment5/assignment5.js

```diff
@@ -0,0 +1,27 @@
+const fs = require('fs');
+
+function readingFile(file) {
+  return new Promise(function (resolve, reject) {
+    fs.readFile(file, 'utf8' , function (err, data) {
+      if (err) {
+        reject(err);
+      }
+      resolve(data);
+      console.log("===========================");
+    })
+  });
+}
+
+readingFile("./f1")
```

Comment:
What if an Error occurs. Have you thought what will happen if your readingFile function breaks, due to some I/O operation.